frontends/main/src/app-pages/DashboardPage/DashboardLayout.tsx

```diff
@@ -77,28 +77,14 @@ const Background = styled.div(({ theme }) => ({
   },
 }))
 
-const Page = styled.div(({ theme }) => ({
```

Comment:
Page is replaced by PageContainer in this PR.

The flex styling is irrelevant here since it only has one child anyway.